environments/storage/gen2/package.json

```
@@ -0,0 +1,8 @@
+{
+  "private": true,
+  "name": "@aws-amplify/ui-gen2-storage-environment",
+  "version": "0.0.1",
+  "scripts": {
+    "generate": "AWS_REGION=us-east-2 npx @aws-amplify/backend-cli@latest generate outputs --branch gen2-storage-backend --app-id d3w3sj6rqvjdb0 --outputs-version 1"
```

I don't think the outputs flag is required anymore as that should be the default now. Also updated the command.
```suggestion
    "generate": "AWS_REGION=us-east-2 npx ampx@latest generate outputs --branch gen2-storage-backend --app-id d3w3sj6rqvjdb0
```

---

Trying to use `ampx` but it does not work. 

```
npm ERR! could not determine executable to run
```

---

OK. Seems Can you just make the outputs-version change? Updated comment above